packages/design-system/src/theme/colors.ts

```diff
@@ -28,6 +28,20 @@ export type MetaColors = ChakraTheme['colors'] & {
   borderPurple: string;
   pinkShadeOne: string;
   secondaryBlue: ColorHues;
+  landing50: string;
```

Comment:
@luxumbra You want to name this color something like `landingPurple`
```suggestion
  landingPurple: ColorHues;
```

Reply:
@dan13ram, I agree with this in part. #1157 is about switching to a color description scheme that is more easily themable by third parties. So instead of using specific color names we would use semantic descriptions like `landing.title` and `landing.text` so that other organizations can replace those colors with their own schemes.

Another useful feature I would like to leverage more are the [`textStyle` and `layerStyle` properties](//chakra-ui.com/docs/features/text-and-layer-styles) that allow the easy reuse of certain styles associated with typography and layout.